components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/AbstractDeviceDAOImpl.java

```diff
@@ -798,9 +798,9 @@ public int getEnrolmentByStatus(DeviceIdentifier deviceId, Status status,
         ResultSet rs = null;
         try {
             conn = this.getConnection();
-            String sql = "SELECT e.ID AS ENROLMENT_ID FROM DM_ENROLMENT e, (SELECT d.ID FROM DM_DEVICE d, DM_DEVICE_TYPE t " +
+            String sql = "SELECT e.ID ENROLMENT_ID FROM DM_ENROLMENT e, (SELECT d.ID FROM DM_DEVICE d, DM_DEVICE_TYPE t " +
```

Comment:
This should be corrected as "SELECT e.ID ENROLMENT_ID" -> "SELECT e.ID AS ENROLMENT_ID"